client/components/TestPlanReportStatusDialog/WithButton.jsx

```diff
@@ -58,7 +59,7 @@ const TestPlanReportStatusDialogWithButton = ({ testPlanVersionId }) => {
             if (matchingReport) {
                 const percentComplete =
                     calculateTestPlanReportCompletionPercentage(matchingReport);
-                if (percentComplete === 100) {
```

Comment:
Nice! Missed that on the button label when [adding the markedFinalAt](https://github.com/w3c/aria-at-app/pull/728/commits/61ce6127e28ffabaa439fa09ba2fd44f0913c5a9) check elsewhere.